client/my-sites/sidebar/style.scss

```
@@ -41,6 +41,7 @@ $brand-text: "SF Pro Text", $sans;
 		// Resetting these colors to prevent bleeding from dashboard color schemes
 		--color-sidebar-text: var(--studio-gray-80);
 		--color-sidebar-text-alternative: var(--studio-gray-40);
+		--color-sidebar-text-alternative-selected: rgba(255, 255, 255, 0.7);
```

I think the previous variable wasn't working because you'd need to suffix it with `-rgb` - this should work?

```suggestion
		--color-sidebar-text-alternative-selected: rgba(var(--studio-white-rgb), 0.7);
```

---

Good call! Thanks @Aurorum.

---

Thanks Aurorum! That does indeed work! I forgot we had that 😅 

Whats weird is both forums and chatGPT say that using this with a hex color should convert to rgb which didn't seem to work, I even went as far as trying to convert the color to an rgb color, but in hindsight that was probably calculating to something like rgba(rgb(... ), 0.7) which is silly. 🤦‍♀️ 

Either way, this works great and sits much better with me than hardcoding the color. Thanks again for pointing this out!